ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -647,6 +647,7 @@ public class BottomSheetController: UIViewController {
                 }
                 animator.startAnimation()
             } else {
+                animator.startAnimation() // moves the animator into active state so it can be stopped
```

Comment:
i am not sure i understand the bug. if the animation was nver started why can we check if it is running and only stop when we need to?

Reply:
The thing we want to achieve in this else branch is to skip to the end of the animation. Because of how `UIViewPropertyAnimator` is designed, to skip to the end you first have to move it to the active state (`startAnimation`), then stop it (`stopAnimation`) and then call finish to ensure all completion handlers are called.

Reply:
do you mind sharing the video of the bug? i still having hard time understnading the issue.

Reply:
There used to be a `pauseAnimation` call at the end of our animator factory, which would move it to active state. That's why the bug didn't repro then.

Reply:
It just crashes when you try to stop an inactive animator. This happens when you for example change the `isExpandable` property after loadView happened, because that causes the sheet to reset to collapsed state.

Reply:
You can actually get a repro by just flipping the "Expandable" switch in the bottom sheet demo controller.

Reply:
Sorry, I should've been more clear in the description :D I added a tldr on top.